webpack.config.js

```diff
@@ -192,6 +191,10 @@ module.exports = (env, argv) => {
       },
     },
     devServer: devServerSettings,
+    watchOptions: {
+      // When running in a container, we can't necesarily watch filesystem events.
+      poll: runInContainer ? true : undefined,
+    },
```

Comment:
I haven't tested this change, but tried to find the right option by looking at documentation.

Reply:
I am (understandably) leery of changes that haven't been tested. Especially config changes XD

Reply:
Well, what is untested in an undocumented development option that appears to have bit-rotted. That it the kind of thing where support is on a best effort basis, and if it breaks for someone, the answer is for them to send a patch to fix it.

Note that only the `runInContainer == true` case in untested, since I'm not exactly sure how to run that, and I don't particularly want to set it up if I did.